tests/models/hubert/test_modeling_tf_hubert.py

```diff
@@ -321,6 +321,20 @@ def test_model_from_pretrained(self):
         model = TFHubertModel.from_pretrained("facebook/hubert-base-ls960")
         self.assertIsNotNone(model)
 
+    # We override here as passing a full batch of 13 samples results in OOM errors for CTC
+    def test_dataset_conversion(self):
```

Comment:
This one, in `TFHubertModelTest`, was the issue. Since it did not have the lower batch size treatment as in the other TF CTC tests, I'm assuming this is the fix.

CI is passing here, but I believe the OOM crash was only happening when a deep change was committed (as it then triggered >1000 tests).